healthyplanet/templates/base.html

```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="h-100">
```

Comment:
Does this need to have the height class or could it be body? or doesn't it matter?

Reply:
The 'h-100' class does need to be on the both root html element and the body element for this technique to work. It's from the example sticky footer method in the bootstrap 5 docs: view-source:https://getbootstrap.com/docs/5.0/examples/sticky-footer/